etc/schema/0.1/ecf/participant_declarations/create/request.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file (and all others) lives in `etc` directory. Is that a typo? If not, what does it stand for?